BEPs/BEP322.md

```diff
@@ -0,0 +1,355 @@
+<pre>
+	BEP: 322
+	Title: Builder API Specification for BNB Smart Chain
+	Status: Draft
+	Type: Standards
+	Created: 2023-11-15
+</pre>
+
+# BEP-322: Builder API Specification for BNB Smart Chain
+
+<!-- TOC -->
+
+- [BEP-322: Builder API Specification for BNB Smart Chain](#bep-322-builder-api-specification-for-bnb-smart-chain)
+  - [1. Summary](#1-summary)
+  - [2. Motivation](#2-motivation)
+  - [3. Background](#3-background)
+    - [3.1 BSC Trust Model](#31-bsc-trust-model)
+    - [3.2 BSC Consensus](#32-bsc-consensus)
+  - [4 Workflow](#4-workflow)
+    - [4.1 Definition](#41-definition)
+    - [4.2 One-Round Interaction (default)](#42-one-round-interaction-default)
+    - [4.3 Two-Round Interaction (optional)](#43-two-round-interaction-optional)
+    - [4.4 APIs](#44-apis)
+      - [4.4.1 Builder APIs](#441-builder-apis)
+        - [4.4.1.1 Issues Report](#4411-issues-report)
+      - [4.4.2 Validator APIs](#442-validator-apis)
+        - [4.4.2.1 Bid Block](#4421-bid-block)
+  - [5. Further Discussion](#5-further-discussion)
+    - [5.1 Implementation Consideration](#51-implementation-consideration)
+    - [5.2 Payment \& Economic Considerations](#52-payment--economic-considerations)
+    - [5.3 Builder Registration](#53-builder-registration)
+  - [6. References](#6-references)
+  - [7. License](#7-license)
+
+<!-- /TOC -->
+
+## 1. Summary
+
+This BEP introduces a suite of interaction processes and API specifications to establish a fairer and more transparent
+MEV framework on the BNB Smart Chain. Participants can follow this specification to collaborate on building and
+proposing blocks.
+
+## 2. Motivation
+
+The Ethereum MEV market is thriving, with
+the [Ethereum Builder API Specification](https://github.com/ethereum/builder-specs&#41) widely supported by
+mainstream consensus clients. As of October 2023, the inclusion rate of blocks from builders is
+approximately [90%](https://dune.com/ChainsightAnalytics/mev-after-ethereum-merge&#41),
+indicating a robust ecosystem.
```

Comment:
It seems have no relationship with`robust ecosystem`

Reply:
removed